about.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Make-up blog</title>
+</head>
+<body>
+    <header>
+        <div class="first-navbar">
+            <a href="https://www.facebook.com"> <img src="./images/facebook_icon.png"> </a>
+            <a href="https://www.facebook.com"> <img src="./images/twitter_icon.png"> </a>
+        </div>
+        <nav>
+            <div class="logo"> <a href="index.html"></a>
+            </div>
+            <div class="nav-bar">
+                <span class="hamburger-icon1">&#9776</span>
+                <span> <a href=""> <img src="./images/logo.jpg" alt=""  class="header-logo" srcset=""></a></span>
+                <span class="cross hidden">X</span>
+            <ul class="second-navbar">
+                <li class="second-navbar-links"><a href="index.html">Home</a></li>
+                <li class="second-navbar-links"><a href="#"></a> Products</li>
+                <li class="second-navbar-links"><a href="#"></a> News</li>
+                <li class="second-navbar-links"><a href="#"></a> login</li>
+            </ul>
```

Comment:
- [ ] This does not show up on the page on desktop/large screens as expected. Kindly ensure that this is fixed